src/components/tx/DecodedTx/index.tsx

```diff
@@ -43,72 +22,63 @@ type DecodedTxProps = {
   decodedData?: DecodedDataResponse
   decodedDataError?: Error
   decodedDataLoading?: boolean
-  showToBlock?: boolean
+  showMethodCall?: boolean
 }
 
 const DecodedTx = ({
   tx,
   txId,
-  showMultisend = true,
   decodedData,
   decodedDataError,
-  decodedDataLoading = false,
-  showToBlock = false,
+  decodedDataLoading,
+  showMultisend = true,
+  showMethodCall = false,
 }: DecodedTxProps): ReactElement | null => {
   const chainId = useChainId()
-  const chain = useCurrentChain()
-  const isSwapOrder = isConfirmationViewOrder(decodedData)
-
   const isMultisend = !!decodedData?.parameters?.[0]?.valueDecoded
+  const isMethodCallInAdvanced = !showMethodCall || isMultisend
 
   const [txDetails, txDetailsError, txDetailsLoading] = useAsync<TransactionDetails>(() => {
     if (!txId) return
     return getTransactionDetails(chainId, txId)
   }, [chainId, txId])
 
-  const addressInfoIndex = txDetails?.txData?.addressInfoIndex
-
   const onChangeExpand = (_: SyntheticEvent, expanded: boolean) => {
     trackEvent({ ...MODALS_EVENTS.TX_DETAILS, label: expanded ? 'Open' : 'Close' })
   }
 
-  if (!decodedData) return null
+  if (!decodedData || !tx) return null
```

Comment:
We cannot return `null` if there is no `decodedData` as this causes the sign modal to show no tx info at all when calling unverified contracts or contracts where we have no ABI.
We should always at least show the `SendToBlock`, `SendAmountBlock` if value > 0 and the raw hex `data`.